Tetris_formal.py

```diff
@@ -156,159 +187,139 @@ def get_direction(self, color):
-            color.blit(color,(self.x, self.y))
-            
-       
+        """
+        同上
+        """
+        color = self.color_or_picture(
+            (20, 90, 90), "/Users/coolguy/Documents/租房照片二.jpg"
+        )
+
         return color
-    
 
 
-class I_block():
-    
-    def __init__(self, x, y):
-        self.x = x
-        self.y = y
-    
-    
+class IBlock(InitializeData):
+    """
+    同上
+    """
+
     def get_direction(self, color):
-       
-        up = [[color, self.x, self.y], [color, self.x, self.y - 40], [color, self.x, self.y + 40], [color, self.x, self.y + 80]]
+        """
+        同上
+        """
+
+        up = [
+            [color, self.x, self.y],
+            [color, self.x, self.y - 40],
+            [color, self.x, self.y + 40],
+            [color, self.x, self.y + 80],
+        ]
         down = up
-        right = [[color, self.x, self.y], [color, self.x - 40, self.y], [color, self.x + 40, self.y], [color, self.x + 80, self.y]]
+        right = [
+            [color, self.x, self.y],
+            [color, self.x - 40, self.y],
+            [color, self.x + 40, self.y],
+            [color, self.x + 80, self.y],
+        ]
         left = right
 
         return up, down, right, left
-    
-    
+
     def get_color(self):
-        num = random.randint(1,2)
-        
-        if num == 1 :
-            color = (70,90,120)
-        
-        elif num == 2 :
-            
-            color = pygame.image.load('/Users/coolguy/Documents/租房照片一.jpg')
-            color = pygame.transform.scale(color,(40,40))
-            color.blit(color,(self.x, self.y))
-            
-       
+        """
+        同上
+        """
+        color = self.color_or_picture(
+            (70, 90, 120), "/Users/coolguy/Documents/租房照片一.jpg"
+        )
+
         return color
-    
 
-    
 
+class OBlock(InitializeData):
+    """
+    同上
+    """
 
-class O_block():
-    
-    def __init__(self, x, y):
-        self.x = x
-        self.y = y
-    
-    
     def get_direction(self, color):
-       
+        """
+        同上
+        """
         up = [
-        [color, self.x, self.y],
-        [color, self.x + 40, self.y],
-        [color, self.x, self.y + 40],
-        [color, self.x + 40, self.y + 40],
+            [color, self.x, self.y],
+            [color, self.x + 40, self.y],
+            [color, self.x, self.y + 40],
+            [color, self.x + 40, self.y + 40],
         ]
         down = up
         right = up
         left = up
 
         return up, down, right, left
-        
-    
+
     def get_color(self):
-        num = random.randint(1,2)
-        
-        if num == 1 :
-            color = (170,90,220)
-        
-        elif num == 2 :
-            
-            color = pygame.image.load('/Users/coolguy/Documents/租房照片四.jpg')
-            color = pygame.transform.scale(color,(40,40))
-            color.blit(color,(self.x, self.y))
-            
-       
-        return color
+        """
```

Comment:
Ditto. The description doesn't mean anything. Please briefly describe the function.